packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -0,0 +1,319 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { TransferDomainKey, TransferDomainType } from '../../../src/category/account'
+import { RpcApiError } from '@defichain/jellyfish-api-core/dist/index'
+import BigNumber from 'bignumber.js'
+
+enum Transfer {
+  THREE = 3
```

Comment:
Do we need to store this on an enum?

Reply:
Since it's just a variable amount

Reply:
just for reading friendly 
eg: https://github.com/BirthdayResearch/jellyfishsdk/blob/a32eab6449c886c5a19cd8abf3eec48f3d62215c/packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts#L57

Reply:
Yep but enum generates it differently on JS. Can just be handled by a simple const/variable I think.

Reply:
removed the enum and just put number directly
https://github.com/BirthdayResearch/jellyfishsdk/pull/2092/commits/8fd524e7424a44e0d327a335a7c614c5def2142a